eap6-support/hawkular-wildfly-agent-feature-pack-eap6/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml

```diff
@@ -616,6 +616,14 @@
         <resource-config-dmr name="Bound Address"
                              path="/socket-binding-group=standard-sockets/socket-binding=http"
                              attribute="bound-address" />
+        <resource-config-dmr name="Immutable"
+                             path="/subsystem=hawkular-wildfly-agent"
```

Comment:
@pilhuhn - are you ok with that behavior? If the WildFly is being remotely monitored (that is, the agent is NOT deployed inside of the wildfly it is monitoring) these values will not be set or be set to null (one of those two, can't remember which).